importCss.js

```diff
@@ -1,32 +1,37 @@
 /* eslint-disable */
 
-var ADDED = {};
+var ADDED = {}
+var NO_CSS_CHUNK_EXPECTED = 'no-css-chunk-expected'
 
 module.exports = function(chunkName) {
   var href = getHref(chunkName)
+
+  if (href === NO_CSS_CHUNK_EXPECTED) {
+    return
+  }
+
   if (!href) {
     if (process.env.NODE_ENV === 'development') {
       if (typeof window === 'undefined' || !window.__CSS_CHUNKS__) {
         console.warn(
           '[DUAL-IMPORT] no css chunks hash found at "window.__CSS_CHUNKS__"'
         )
-        return
+      } else {
```

Comment:
I think this is slightly less readable, but I switched it because CI complained about too many return statements